canonical-web-and-design/ubuntu.com
# Telco page

## Done

- Build telco page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit)
- Check navbar links go to the correct URL (some URL don't have endpoints yet)
- Marketo form is being built in another issue so will not work yet.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4188
